client/my-sites/customize/main.jsx

```diff
@@ -115,8 +115,11 @@ class Customize extends React.Component {
 		return false;
 	};
 
+	getReturnUrl = () => new URLSearchParams( window.location.search ).get( 'return' );
```

Comment:
This is testing well for me manually now, but I think we should add some validation here. Can we [re-use `validateUrl`](https://github.com/automattic/wp-calypso/blob/4537033c6794449ce0a10b69d72daf5fa658d553/client/blocks/login/continue-as-user.jsx#L25) in this function, and if it fails validation, return `null`?

Reply:
Actually, I see that function isn't exported, but we might be able to copy its internals.

Reply:
Updated to use the same API call as in https://github.com/automattic/wp-calypso/blob/4537033c6794449ce0a10b69d72daf5fa658d553/client/blocks/login/continue-as-user.jsx#L25, but didn't import the same function so that we're not pulling in code from the login block.